lib/content-services/src/lib/dialogs/library/library.dialog.ts

```diff
@@ -198,6 +199,8 @@ export class LibraryDialogComponent implements OnInit, OnDestroy {
     }
 
     private handleError(error: any): any {
+        let errorMessage = 'CORE.MESSAGES.ERRORS.GENERIC';
```

Comment:
Please check the 'GENERIC' key is present in the en.json file.

Reply:
https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/core/src/lib/i18n/en.json